android-components/components/service/contile/src/test/java/mozilla/components/service/contile/ContileTopSitesProviderTest.kt

```diff
@@ -70,15 +70,15 @@ class ContileTopSitesProviderTest {
     fun `GIVEN a 500 status response WHEN top sites are fetched THEN throw an exception`() =
         runTest {
             val client = prepareClient(status = 500)
-            val provider = ContileTopSitesProvider(testContext, client)
+            val provider = ContileTopSitesProvider(testContext, false, client)
```

Comment:
I would like to see a test where we do fetch a JSON with this "exclusion" list and ensure we are filtering correctly.